templates/distribution/scripts/delete.sh.tpl

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env sh
+
+kustomizebin="{{ .paths.kustomize }}"
+kubectlbin="{{ .paths.kubectl }}"
+
+if [ "$1" = "--dry-run=true" ]; then
+  dryRun="--dry-run=client"
+else
+  dryRun=""
+fi
+
+$kubectlbin delete --all-namespaces --wait --timeout=180s --all ingress $dryRun
```

Comment:
Maybe the deletion should be the same between Distro and EKS provider, and we should:

* kustomize build everthing, skip CRDs and delete all the CustomResources
* Delete all statefulsets from the build
* Delete all the PVCs from our distro namespaces (logging, monitoring)
* Delete all the svc type LoadBalancers (filtered with yq)
* Delete everything else BUT CRDs
* Delete everything

(using --ignore-not-found on kubectl delete)